src/index.js

```diff
@@ -1,12 +1,55 @@
-const propertiesToConvert = {
-  paddingLeft: 'paddingRight',
-  paddingRight: 'paddingLeft',
-  marginLeft: 'marginRight',
-  marginRight: 'marginLeft',
+// this will be an object of properties that map to their corresponding rtl property (their doppelganger)
+const propertiesToConvert = arrayToObject([
+  ['paddingLeft', 'paddingRight'],
+  ['marginLeft', 'marginRight'],
+  ['left', 'right'],
+])
+
+// this is the same as the propertiesToConvert except for values
+const valuesToConvert = arrayToObject([
```

Comment:
Should these values be scoped to specific properties? I guess I don't know if 'left' would ever be a valid value of some property that we wouldn't want to switch...